engine/src/globals.cpp

```diff
@@ -1266,6 +1266,9 @@ bool X_open(int argc, MCStringRef argv[], MCStringRef envp[])
 
 int X_close(void)
 {
+    // MW-2012-02-23: [[ FontRefs ]] Finalize the font module.
+    MCFontFinalize();
```

Comment:
Hmmm - looks like some of these Finalize calls are duplicated here and lower down...